### DISCUSSION TITLE
How to add a clickable button and a keyboard response

### DISCUSSION THREAD

**Discussion by drwecki** [jspsych/jsPsych#745] [Q&A]
I'm a bit of a newb, but, I've programed in JS before. I've been messing around with the IAT plug in I want to alter the plug-in to have.a permanent mouse button that allows the user to skip the trial. While, I've created the button and can virtually click the button in dev tools console [setting a new variable equal to my button click action in the document], everything works the way I want, but I can't  click the button with my cursor [I can see it; I can't physically click it]  and don't have a strong enough background to figure out why the little hand doesn't show up to allow me to click. I'm guessing The HTML IAT plug=in locks out mouse events directly?  Any help would be appreciated.

**Comment by vijaymarupudi** [comment]
Can you posting code of the modifications and the jsPsych code you're trying to run?

**Reply by vijaymarupudi** [reply]
I'd like to copy paste this and analyze it, could you format it correctly using this link <https://help.github.com/en/github/writing-on-github/creating-and-highlighting-code-blocks>

I would do it, but I don't seem to have the permissions to view the raw markdown in your post.

**Reply by drwecki** [reply]
I got it, thanks for the help.

**Reply by aliusama980** [reply]
Can you please share the clickable button code?
